### ISSUE TITLE
NBD Server mounts drive, but not detecting it

### ISSUE THREAD

**Issue by TheMrIron2** [GDX-X/PFS-BatchKit-Manager#56]
I followed the instructions in "How to Setup NBD Server (Network)" in the README.
When I reached the last step, I heard the Windows sound effect that indicates a new drive has connected... but it does not appear as a local drive, and PFS Manager does not detect the drive.

It _does_ appear in "Show list of mounted devices":
![image](https://github.com/GDX-X/PFS-BatchKit-Manager/assets/12585537/c08b5cb2-9c3e-4252-b85a-34bee4d899b7)

And in Windows Device Manager, as an "Unallocated" drive:
![image](https://github.com/GDX-X/PFS-BatchKit-Manager/assets/12585537/58fc880f-6da5-4b2a-85cd-3cd2c1a3a732)

However, it does not show up in File Explorer, and PFS Manager still says "[NO DEVICE] PLEASE RELOAD HDD". Trying to rescan HDDs at the main menu reports "Playstation 2 HDD Not Detected / Drive Must Be Formatted First".

TL;DR I followed the instructions and connected via NBD, but PFS is not detecting the PS2 HDD.
